### ISSUE TITLE
Do we need two npm installs?

### ISSUE THREAD

**Issue by michaelpporter** [xenomedia/xeno_robo#1]
One for grunt and one for any site global settings?

i.e. execadv as a core npm for the git commit checks.

**Comment by albertski** [comment]
Good point. Maybe we should have a config field called npm_paths and have the ability to have multiple.
